### PR TITLE
Drichards sla endpoint

### PR DESCRIPTION
**Pull request by dpricha89**
Added a http endpoint to monitor an applications SLA status.

How it works:
1. Message comes in from Kafka
2. The message is parsed and the TS is sent to SlaClient.checkTS()
3. The timestamp is older than the allowed SLA
4. The Boolean _isSlaMet is updated to `false` value
5. The monitoring server is using <servername>/__sla endpoint to look for non `true` values and alerts pagerduty

### CODE REVIEW

**Comment by jeffallen6767** [comment]
can you take me off this repo please? thx

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by hanpeter** [comment]
@jeffallen6767: You are probably watching this repo. This is an open source repo. That's probably why you still can watch this repo. Feel free to unwatch it. 😃

**Comment by jeffallen6767** [comment]
Thanks Peter, I'll un-watch.


________________________________

**Comment by weizhu-us** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -49,6 +49,31 @@ public class ExampleMain {
...
+
+Example from a consumer service sending message timestamps to the `SlaClient` for verification.
+
+```java
+import com.krux.stdlib.utils.SlaClient;
+
+public class messageProcessor { 
+    
+    private static SlaClient _slaClient = SlaClient.getInstance();
+
```
Comment (weizhu-us) [inline README.md]: @dpricha89. I don't need to initialize SlaClient here, do I?

**Pull request merged**